NuGet.Config

```diff
@@ -54,12 +59,102 @@
+      <package pattern="Microsoft.DataAI.NuGetRecommender.Contracts" />
+      <package pattern="Microsoft.DiaSymReader" />
+      <package pattern="Microsoft.DotNet.Build.Tasks.Feed" />
+      <package pattern="Microsoft.DotNet.Maestro.Tasks" />
+      <package pattern="Microsoft.Extensions.FileProviders.Abstractions" />
+      <package pattern="Microsoft.Extensions.FileSystemGlobbing" />
+      <package pattern="Microsoft.Extensions.Primitives" />
       <package pattern="microsoft.internal.visualstudio.*" />
+      <package pattern="Microsoft.Internal.VisualStudio.Shell.Framework" />
+      <package pattern="Microsoft.IO.Redist" />
+      <package pattern="Microsoft.Net.Compilers.netcore" />
+      <package pattern="Microsoft.Net.Compilers.Toolset" />
+      <package pattern="Microsoft.NET.StringTools" />
+      <package pattern="Microsoft.NET.Test.Sdk" />
+      <package pattern="Microsoft.NETCore.Platforms" />
+      <package pattern="Microsoft.PowerShell.3.ReferenceAssemblies" />
       <package pattern="microsoft.servicehub.*" />
+      <package pattern="Microsoft.ServiceHub.Framework" />
+      <package pattern="Microsoft.TeamFoundationServer.ExtendedClient" />
       <package pattern="microsoft.test.apex.visualstudio" />
+      <package pattern="Microsoft.TestPlatform.Portable" />
+      <package pattern="Microsoft.TestPlatform.TestHost" />
       <package pattern="microsoft.visualstudio.*" />
+      <package pattern="Microsoft.VisualStudio.ComponentModelHost" />
+      <package pattern="Microsoft.VisualStudio.LanguageServices" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Interop" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Managed" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.Managed.VS" />
+      <package pattern="Microsoft.VisualStudio.ProjectSystem.VS" />
+      <package pattern="Microsoft.VisualStudio.SDK" />
+      <package pattern="Microsoft.VisualStudio.Sdk.TestFramework.Xunit" />
+      <package pattern="Microsoft.VisualStudio.Setup.Configuration.Interop" />
+      <package pattern="Microsoft.VisualStudio.Shell.15.0" />
+      <package pattern="Microsoft.VisualStudio.VCProjectEngine" />
+      <package pattern="Microsoft.VisualStudio.Workspace.VSIntegration" />
+      <package pattern="Microsoft.VisualStudioEng.MicroBuild.Core" />
+      <package pattern="Microsoft.VSSDK.BuildTools" />
+      <package pattern="Microsoft.Web.Xdt" />
+      <package pattern="Microsoft.Win32.Registry" />
+      <package pattern="Moq" />
+      <package pattern="MSTest.TestAdapter" />
+      <package pattern="MSTest.TestFramework" />
+      <package pattern="Newtonsoft.Json" />
+      <package pattern="NuGet.Client.EndToEnd.TestData" />
+      <package pattern="NuGet.Core" />
+      <package pattern="NuGetValidator" />
+      <package pattern="Portable.BouncyCastle" />
+      <package pattern="SharpZipLib" />
```

Comment:
NuGet.Client.EndToEnd.TestData and NuGetValidator has close to zero percent chance of being on the vside feed. There's no way vside (well, msft_consumption) upstreams from our nuget-build feed, and this package should only exist in that nuget-build feed. This pattern already exists for the nuget-build feed, so I'm scratching my head why you needed to add the same pattern to the vside feed.

The others are not even authored by Microsoft (well MSTest is, but I feel like the justification is the same), so it just seems infeasible that there are VS specific versions, so the mapping that already exist for the dotnet-public feed should suffice. It's certainly not clear to me why these mappings were added to vside without removing them from dotnet-public.